doc/Yang Model Guidelines.md

```diff
@@ -250,4 +244,5 @@ Example:
 ```
 
 
-## All above steps are manual right now, in future few of them can be automated.
+## All above steps are manual right now, in future few of them can be automated. Below tasks are automated already:
```

Comment:
I believe this is not a guideline step and should be moved to a different section.

Reply:
I would like to drop these, as this YANG is where we put the schema  and ***the primary source*** for all other requirements. Like CVL YANG is generated from this using a build time tool. The header files would be generated off this.

I really don't see scope for this YANG getting auto generated, which would imply one another source for schema, which is none in my knowledge.